packages/api-server/api_server/app.py

```diff
@@ -66,22 +66,23 @@ async def on_sio_connect(sid: str, _environ: dict, auth: Optional[dict] = None):
     allow_methods=["*"],
     allow_headers=["*"],
 )
-os.makedirs(app_config.static_directory, exist_ok=True)
+
 app.mount(
     "/static",
-    StaticFiles(directory=app_config.static_directory),
+    StaticFiles(directory="static"),
```

Comment:
```suggestion
    StaticFiles(directory=f"{__dir__}/static"), # may not be correct, check with the output `whl`
```
The directory need to be relative to the package's dir (e.g. when release in ros it will be some form of `/opt/ros/iron/lib/python-packages/...`.

Reply:
Yup good call, using the absolute path of `app.py` works